service/location/internal/endpoints/endpoints.go

```diff
@@ -127,7 +127,7 @@ var defaultPartitions = endpoints.Partitions{
 			{
 				Variant: 0,
 			}: {
-				Hostname:          "geo.{region}.amazonaws.com",
```

Comment:
got this one from aws-cli (using `--debug` flag), i'm not sure about the rest of the urls in this file though (geo-fips, geo.{region}.api, etc).